### ISSUE TITLE
Create doesn't work

### ISSUE THREAD

**Issue by chalkers** [matrix-io/matrix-cli#14]
The create command doesn't work (windows / Linux)

`matrix create nfc`

Windows: 

```
An error occurred: { Error: ENOENT: no such file or directory, open 'C:\Users\andre\Projects\nfc\:'
    at Error (native)
  errno: -4058,
  code: 'ENOENT',
  syscall: 'open',
  path: 'C:\\Users\\andre\\Projects\\nfc\\:' }
```

Linux I get a similar issues.

**Comment by eighteyes** [comment]
Ok, thanks. Was able to reproduce. I'll look into this.

**Comment by eighteyes** [comment]
Error on *nix
```
Error: EACCES: permission denied, mkdir '/usr/local/lib/node_modules/matrix-cli/lib/../tmp'
```

**Comment by eighteyes** [comment]
Fixed f913007
